brayns/engine/components/ClippingComponent.h

```diff
@@ -36,43 +33,27 @@ template<typename T>
 class ClippingComponent : public Component
 {
 public:
-    ClippingComponent(const T &geometry)
-    {
-        _geometry.add(geometry);
-    }
-
-    Geometry<T> &getGeometry() noexcept
+    ClippingComponent(const T &primitive)
+        : _clipper(std::move(primitive))
```

Comment:
You call move but you make a copy as primitive is const, which one is intended ?

Reply:
Addressed, nice catch!